src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -99,4 +100,17 @@
         public static final int XBOX_PORT = 0;
         public static final double RUMBLE_INTERMISSION_TIME = 0.15;
     }
+
+    public static class IntakeConstants {
+        public static final TrigonTalonSRX MOTOR = IntakeComponents.MOTOR;
+    }
+
+    public static class IntakeOpenerConstants {
+        public static final TrigonTalonSRX MOTOR = IntakeOpenerComponents.MOTOR;
+        public static final double OPERATING_POWER = 0.4;
```

Comment:
what is this value?

Reply:
3 but i see that there is a problem so can you say more about it

Reply:
o you mean oporating power me dumb

Reply:
ill delete it